_conferences/devcommunity-summit-2021.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "DevCommunity Summit 2021"
```

Comment:
Do you want to mention "Previously Android Summit" in the name as well?

Reply:
Good call, will update!